Entity/Form.php

```diff
@@ -100,6 +100,7 @@ public function getTranslation(string $locale, bool $create = false, bool $fallb
             $translation = new FormTranslation();
             $translation->setLocale($locale);
             $this->addTranslation($translation);
+            $translation->setForm($this);
```

Comment:
the same thing is already done in the `FormField`:
https://github.com/sulu/SuluFormBundle/blob/1ab810f829233c6c006c2bac74406aead7a68137/Entity/FormField.php#L167